File: src/evalarc/assets/robot_starter.js
// Implement the recording-review contract described in TASK.md.
const readline = require("node:readline");

function review(request) {
  throw new Error("Convert recorded observations and return the requested facts");
}

const lines = readline.createInterface({ input: process.stdin, crlfDelay: Infinity });
lines.on("line", (line) => {
  process.stdout.write(JSON.stringify({ ok: true, report: review(JSON.parse(line)) }) + "\n");
});

File: examples/behavior-audit/pilot/harness/evalarc/src/evalarc/assets/robot_starter.py
"""Implement the recording-review contract described in TASK.md."""

import json
import sys


def review(request):
    raise NotImplementedError("Convert recorded observations and return the requested facts")


for line in sys.stdin:
    print(json.dumps({"ok": True, "report": review(json.loads(line))}), flush=True)
